doc/introduction/chemistry.rst

```diff
@@ -153,26 +85,24 @@ expectation value of a Hamiltonian can be calculated using ``qml.expval``:
 .. code-block:: python
 
     import pennylane as qml
+    from pennylane import qchem
+    from pennylane import numpy as np
 
     dev = qml.device('default.qubit', wires=4)
 
-    hamiltonian = 2.0 * qml.PauliZ(0) @ qml.PauliZ(1)
+    symbols, coordinates = (['H', 'H'], np.array([0., 0., -0.66140414, 0., 0., 0.66140414]))
+    hamiltonian, qubits = qchem.molecular_hamiltonian(symbols, coordinates)
 
     @qml.qnode(dev)
     def circuit(params):
-        qml.BasisState(np.array([1, 1, 0, 0]), wires=[0,1,2,3])
-        for i in range(4):
-            qml.Rot(*params[i], wires=i)
-        qml.CNOT(wires=[2, 3])
-        qml.CNOT(wires=[2, 0])
-        qml.CNOT(wires=[3, 1])
+        qml.BasisState(np.array([1, 1, 0, 0]), wires=[0, 1, 2, 3])
+        qml.DoubleExcitation(params, wires=[0, 1, 2, 3])
         return qml.expval(hamiltonian)
 
-    rng = np.random.default_rng(seed=42)
-    params = rng.random([4, 3])
+    params = np.array(0.0, requires_grad=True)
     circuit(params)
 
-The rotation angles can be optimized using the machine learning interface of choice
+The circuit parameter can be optimized using the machine learning interface of choice
```

Comment:
I think we can simply mention `interface` instead of `machine learning interface`.